### PR TITLE
Add windows support

### LINKED ISSUE #2: Port to Windows
Same as #1 , but for windows.

See [W10 Toast Notifications](https://github.com/jithurjacob/Windows-10-Toast-Notifications)

### PR DESCRIPTION
**Pull request by monkeek**
Closes #2

### CODE REVIEW

File: etnotify.py [hunk 1]
```diff
@@ -21,14 +27,30 @@ def get_logger():
...
-    notifier.show()
+    if platform == "linux":
+        notify2.init('etnotif')
+    else:
+        notifier = ToastNotifier()
+    send_notification(notifier, "ETNA", 'initialized etnotify')
     return notifier
 
 
+
```
Comment (tbobm) [inline etnotify.py]: Could you remove this blank line ?

File: etnotify.py [hunk 2]
```diff
@@ -5,7 +5,13 @@
 
 from etnawrapper import EtnaWrapper
 from etnawrapper import BadStatusException
-import notify2
+
+try:
+    import notify2
+    platform = "linux"
```
Comment (tbobm) [inline etnotify.py]: Maybe this would be a bit clearer if the `platform` variable was uppercase, as it looks like a constant

**Review by tbobm — CHANGES_REQUESTED** [review]

File: etnotify.py (hunk 1 above)
Comment (monkeek) [reply etnotify.py]: Done :)

File: etnotify.py (hunk 2 above)
Comment (monkeek) [reply etnotify.py]: Done ;)

**Review by tbobm — APPROVED** [review]

**Pull request merged**